Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml

```diff
@@ -52,7 +52,9 @@
   - type: Item
     sprite: Objects/Tools/spy_device.rsi
   - type: PowerCellDraw
-    useRate: 40 #Should be weaker than the original
+    useRate: 10
+  - type: StaticPrice
+    price: 750
```

Comment:
uh why?

Reply:
a regular handheld crew monitor costs 500. This one is an improved version that uses less energy and is not easy to get for sale at the cargo. Why not?

Reply:
Why should a use rate change be in a PR adding a new antagonist?